### DISCUSSION TITLE
每次更新文章/網站setting的時候都會把文章和網站的統計數據reset

### DISCUSSION THREAD

**Discussion by gaminglifex** [Qexo/Qexo#82] [Q&A]
每次更新文章/網站setting的時候都會把文章和網站的統計數據reset
![image](https://user-images.githubusercontent.com/72642816/153158121-d3c5b26e-42a5-41a7-b75d-a9a37ce8f5ae.png)
應該是Created on: 2022-02-03 Updated on: 2022-02-09 才對吧?
我正在結合Qexo+Hexo+Butterfly主題使用
應該是哪個位置出錯? :))

**Comment by am-abudu** [comment]
我想Qexo或许不会修改这些东西，请问这两个值是如何存储的呢

**Reply by gaminglifex** [reply]
至於兩個值是如何存儲 應該是根據這個設定
http://busuanzi.ibruce.info
![image](https://user-images.githubusercontent.com/72642816/153694548-7ea17dc2-63d8-486b-8fa1-7924487205c5.png)

**Reply by am-abudu** [reply]
> 至於兩個值是如何存儲 應該是根據這個設定 http://busuanzi.ibruce.info ![image](https://user-images.githubusercontent.com/72642816/153694548-7ea17dc2-63d8-486b-8fa1-7924487205c5.png)

不是很清楚，你说的是Created on的时间还是访问量统计，我的猜想如下：
如果是Created on的时间，那么这个Qexo不会进行更改，请检查相关配置
如果是访问量统计，应该是基于网页链接进行统计的，Qexo依然不会对此进行修改，检查Hexo配置中的permalink项吧:)

**Comment by Fgaoxing** [comment]
没碰到过，请问网址是多少？

**Reply by gaminglifex** [reply]
我blog的網址暫時設定在 
https://gaminglifex-blog.vercel.app

**Comment by Fgaoxing** [comment]
@gaminglifex 可以尝试改用LeanCloud吗？
